### PR TITLE
Backend and frontend communication

### PR DESCRIPTION
**Pull request by Filip-B-32**
Made communication between backend and frontend (+CORS)
CRUD on backend (example CRUD)
CRUD request on frontend (working and made with axios)

### CODE REVIEW

**Comment by Filip-B-32** [comment]
i added a .env in frontend with a local host example. Each one will modify the local host port according to our own port
we will not push the .env file but i pushed this time to see it as an example we have to follow

**Review by alexandratodea — APPROVED** [review]
looks good :)

File: EventPlannerAPI/EventPlannerAPI/Controllers/UserController.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+namespace EventPlannerAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UserController : ControllerBase
+    {
+        public UserController() { }
+
+        [HttpGet("/getUsers")]
```
Comment (tividan2014) [inline EventPlannerAPI/EventPlannerAPI/Controllers/UserController.cs:13]: I think, you can remove verb from name

**Review by tividan2014 — APPROVED** [review]

**Review by gheorghe-sava — APPROVED** [review]
Looks good

**Pull request merged**